src/main/java/com/skcraft/plume/module/Status.java

```diff
@@ -0,0 +1,104 @@
+package com.skcraft.plume.module;
+
+import com.sk89q.intake.Command;
+import com.sk89q.intake.Require;
+import com.sk89q.intake.parametric.annotation.Optional;
+import com.sk89q.intake.parametric.annotation.Switch;
+import com.skcraft.plume.command.Sender;
+import com.skcraft.plume.common.util.SharedLocale;
+import com.skcraft.plume.common.util.module.Module;
+import com.skcraft.plume.event.entity.EntityEvent;
+import com.skcraft.plume.util.Messages;
+import com.skcraft.plume.util.Server;
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.util.ChatComponentText;
+import net.minecraftforge.event.entity.living.LivingAttackEvent;
+
+import javax.swing.text.html.parser.Entity;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import static com.skcraft.plume.common.util.SharedLocale.tr;
+
+@Module(name = "status", desc = "Heal, feed and god commands")
+public class Status {
+
+    private final Set<UUID> gods = new HashSet<>();
+
+    @Command(aliases = "heal", desc = "Regain full health points")
+    @Require("plume.status.heal")
+    public void heal(@Sender ICommandSender sender, @Optional EntityPlayerMP target, @Switch('a') Integer amount) {

```

Comment:
You can just do `@Sender EntityPlayerMP` so you wouldn't have to check if the sender is a player.

Ditto for the other methods.